Library/Homebrew/formula.rb

```diff
@@ -310,7 +310,12 @@ def full_installed_alias_name
 
   # The path that was specified to find this formula.
   def specified_path
-    alias_path || path
+    default_specified_path = alias_path || path
+
+    return default_specified_path if default_specified_path.presence&.exist?
```

Comment:
`alias_path` is a String rather than a Pathname, so this can error in that scenario.

I'm tempted to make `alias_path` a Pathname but that would be an API change.

Reply:
```suggestion
    default_specified_path = Pathname(alias_path) || path

    return default_specified_path if default_specified_path.presence&.exist?
```

This will remove the error possibility, right?

Reply:
`Pathname(nil)` is an error unfortunately.